src/System.Private.ServiceModel/tools/setupfiles/CleanupWCFTestService.cmd

```diff
@@ -3,11 +3,17 @@ setlocal
 
 echo BridgeKeepRunning=%BridgeKeepRunning%
 if '%BridgeKeepRunning%' neq 'true' (
-    echo Stopping the Bridge...
+    echo Releasing Bridge resources and stopping it if running locally.
     pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    call Bridge.exe -stopIfLocal %*
+    echo Invoking Bridge.exe -stopIfLocal -reset %* ...
+    call Bridge.exe -stopIfLocal -reset %*
     popd
 ) else (
+    echo Releasing Bridge resources but leaving it running.
+    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
+    echo Invoking Bridge.exe -reset %* ...
+    call Bridge.exe -reset %*
+    popd
     echo The Bridge was left running because BridgeKeepRunning is true
```

Comment:
IIRC, CleanupWcfTestService.cmd is called immediately after a run happens. Do we necessarily want to get rid of all resources that the bridge is holding? 

My flow of work is typically the following: 
- run startBridge.cmd
- run build.cmd to look at what's wrong
- make changes
- rebuild. 

Sometimes, in the "rebuild" step I run build.cmd, other times I build the project and test directly. 
My suspicion is that if someone has left the bridge running, we should probably leave any resources opened alone, and rely on the developer to either `exit` the Bridge or call `reset` manually